source/back.tex

```diff
@@ -52,11 +52,11 @@
   \doccite{The \Cpp{} Programming Language, second edition}, Chapter R\@.
   Addison-Wesley Publishing Company, ISBN 0-201-53992-6, copyright \copyright 1991 AT\&T
 \bibitem{kr}
-  Brian W.\ Kernighan and Dennis M. Ritchie,
+  Brian W.\ Kernighan and Dennis M.\ Ritchie,
   \doccite{The C Programming Language}, Appendix A\@.
   Prentice-Hall, 1978, ISBN 0-13-110163-3, copyright \copyright 1978 AT\&T
 \bibitem{cpp-lib}
-  P.J.\ Plauger,
+  P.\,J.\ Plauger,
   \doccite{The Draft Standard \Cpp{} Library}.
   Prentice-Hall, ISBN 0-13-117003-1, copyright \copyright 1995 P.J.\ Plauger
```

Comment:
If you're going to change L59, then also here?

Reply:
Done; thanks! : - )